backend/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "backend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Just a suggestion, adding a script called `start` is the idiomatic/preferred way in Node to start the application. Add sone like,
```json
    "start": "node src/index.js"
```

After this, you can simply run `npm run start` or `yarn start` to run the server.

Reply:
On it sir

Reply:
Sir, I had a doubt regarding port. I've heard that ports above 3000 are generally used, while the port 3000 is itself reserved for React. Is there any such significance or good practice to use a certain port number for certain things?

Reply:
There isn't. You can use any port that you find easy to remember. As with CRA, the default one is `3000`, we avoid using that in backend code.